<launch>
  <group ns="ic120">
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

      <rosparam file="$(find ic120_nav)/params/costmap_common_params.yaml" command="load" ns="global_costmap" />
      <rosparam file="$(find ic120_nav)/params/costmap_common_params.yaml" command="load" ns="local_costmap" />

      <rosparam file="$(find ic120_nav)/params/odom_nav_params/global_costmap_params.yaml" command="load" />
      <rosparam file="$(find ic120_nav)/params/odom_nav_params/local_costmap_params.yaml" command="load" />

      <rosparam file="$(find ic120_nav)/params/base_local_planner_params.yaml" command="load" />
      <rosparam file="$(find ic120_nav)/params/move_base_params.yaml" command="load" />

      <param name="base_global_planner" type="string" value="navfn/NavfnROS" />
      <param name="base_local_planner" value="base_local_planner/TrajectoryPlannerROS"/>


      <!-- <remap from="odom" to="odometry/filtered" /> -->
    </node>
  </group>
 </launch>